rust-src/concordium_base/src/eddsa_ed25519/ffi.rs

```diff
@@ -56,7 +56,7 @@ extern "C" fn eddsa_public_to_bytes(
 #[no_mangle]
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
 extern "C" fn eddsa_public_from_bytes(input_bytes: *mut u8, input_len: size_t) -> *mut PublicKey {
-    let len = input_len as usize;
+    let len = input_len;
```

Comment:
As in the other ffi, we should probably retain the case from `size_t` to `usize` in case they are different.